Dockerfile

```diff
@@ -32,7 +32,7 @@ RUN apt-get update \
     make \
     openssh-server \
     openssl \
-    python3-pip \
+    python3-jinja2 \
```

Comment:
Newer versions of Python prevent `pip` from installing to [externally managed environments](https://packaging.python.org/en/latest/specifications/externally-managed-environments/), so install the package via `apt`.